script.module.future/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.future" name="future" version="0.16.0" provider-name="Ed Schofield">
+<addon id="script.module.future" name="future" version="0.16.0.1" provider-name="Ed Schofield">
```

Comment:
Don't get me wrong, I prefer x.y.z...
I just wanted to prevent the "future module has a new version 0.16.1, now we can't keep the version number in sync".
The additional .1 is the best I could come up with :P